packages/dendron-next-server/styles/theme.ts

```diff
@@ -0,0 +1,34 @@
+import { theme as defaultTheme, extendTheme } from "@chakra-ui/react";
+
+export const theme = extendTheme({
+  config: {
+    useSystemColorMode: true,
+  },
+
+  colors: {
+    positive: defaultTheme.colors.green,
+
+    brand: {
+      // Generated with https://copypalette.app/ via #69B160
```

Comment:
Not sure what the official color of Dendron is, just eyedropped from the logo. LMK if wrong.

(also a related suggestion to add to the main `dendron.so` site: a `Dendron Branding` page that has the logo, brand color/s, etc.)

Reply:
I've added `dendron.topic.style` to the dendron-site: https://github.com/dendronhq/dendron-site/blob/master/vault/dendron.topic.style.md

It's currently hidden from the webpage because it's rather sparse but feel free to start adding to it when it makes sense :)

Reply:
Thanks for starting the note -- here's something super-simple to kick that note off: https://github.com/dendronhq/dendron-site/pull/41